operations/service.go

```diff
@@ -49,6 +49,7 @@ func parseDB(c *cli.Context) *evergreen.DBSettings {
 	if url == evergreen.DefaultDatabaseUrl && envUrl != "" {
 		url = envUrl
 	}
+	envAuth := os.Getenv(evergreen.MongodbAuthFile)
```

Comment:
should this be a command line option as well?

Reply:
I think that complicates things as there are multiple command line entry points. I like the simpler interface: if the environment variable is set, this is the auth file, and if it isn't, there is no auth file.

Reply:
Here too [for cedar we used a different approach](https://github.com/evergreen-ci/cedar/blob/21fcb86a728e54a3bad1b2cf72535f5da01ee58d/operations/flags.go#L123-L126): if it's set on the CLI then use that. Otherwise use the environment. This would give more flexibility to run the application as you want no matter what's in the environment.

This seems like a common paradigm (e.g. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html))

Reply:
I think I misunderstood what the cedar code was doing when I wrote it. That makes sense.